### ISSUE TITLE
A11y_Azure Data Studio_Query Plan (Preview)_ScreenReader: NVDA announcing as graphic link for the images present in tree view

### ISSUE THREAD

**Issue by poojapatil36** [microsoft/azuredatastudio#20777]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #ScreenReader; #WCAG4.1.2; #FTP; #NVDA

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:

1. Launch the "Azure data studio" application.
2. Connect with the provided SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. Observe that NVDA announcing as graphic link for the images present in tree view.

### Actual:
NVDA announcing as graphic link for the images present in tree view. The graphic present in tree view is not actionable and not working as link.

### Expected Result:
NVDA should not announce as graphic link for the images present in tree view. 

### Attachment:
![A11y_Azure Data Studio_Query Plan (Preview)_ScreenReader (3)](https://user-images.githubusercontent.com/97668775/194810822-0edcae1a-bf50-41d4-90e7-4fdf820e7395.gif)

**Comment by aasimkhan30** [comment]
@poojapatil36 , it seems NVDA is only announcing graphic when the focus first enters the chart. Which is expected. It does not announce graphic when we are navigating between nodes.

**Comment by aasimkhan30** [comment]
![azuredatastudio_13YXztU2u6](https://user-images.githubusercontent.com/6816294/195776049-d0f398a4-380e-45f8-8d97-93e948235c17.gif)

**Comment by poojapatil36** [comment]
@aasimkhan30 For second node also it is announced as link for graphic when navigate the tree view in scan mode.

**Comment by aasimkhan30** [comment]
@poojapatil36 , this issue is fixed in the latest insiders. Can you please test it?

**Comment by Rama8345** [comment]
@aasimkhan30 
Verified the bug in the latest insiders build, but we are getting below error.
![image](https://user-images.githubusercontent.com/115216656/199224780-816f3427-3543-4880-81c9-bf186522c6d7.png)

**Comment by Rama8345** [comment]
Verified the issue on latest build, now we are unable to navigate the tree view in scan mode. Now when we are pressing "Insert + Space" then tooltip is appearing. Please find attachment for reference.
![20777_NotFixed](https://user-images.githubusercontent.com/115216656/199437641-502a706e-9297-4a9a-9a0e-98ce57a155b7.gif)

**Comment by poojapatil36** [comment]
Verified the issue in latest build, now we are unable to navigate tree view in scan mode. Please refer below attachment.
![Presentation4](https://user-images.githubusercontent.com/97668775/204788695-9a9ae52a-c2b0-4c56-973e-f2dc32b98ae8.gif)

**Comment by poojapatil36** [comment]
@aasimkhan30 Could you please share ETA for fix.

**Comment by Ashutoshsahoo164** [comment]
Hi @aasimkhan30 Is there any ETA for the Fix? Or could you please provide the build number or link where we can get the Fix?

CC : @poojapatil36

**Comment by aasimkhan30** [comment]
@poojapatil36  and @Ashutoshsahoo164  can you please verify the fix?

**Comment by poojapatil36** [comment]
@aasimkhan30 Still I am unable to navigate tree view in scan mode.

**Comment by poojapatil36** [comment]
@aasimkhan30 Could you please let us know the Azure data studio build in which the issue is fixed.

**Comment by poojapatil36** [comment]
Verified the issue on below build.
 Version 1.47.1.
The issue is still repro as we are unable to navigate graph controls in scan mode

**Comment by kisantia** [comment]
@poojapatil36 a fix has been made for this. Please try the latest insiders build, which can be found here: https://github.com/microsoft/azuredatastudio?tab=readme-ov-file#try-out-the-latest-insiders-build-from-main-branch

**Comment by kisantia** [comment]
@poojapatil36 Please try the latest insiders build, which can be found here: https://github.com/microsoft/azuredatastudio?tab=readme-ov-file#try-out-the-latest-insiders-build-from-main-branch

**Comment by poojapatil36** [comment]
@kisantia Could you please help us with the login credentials over protected mail or over teams. so that we can verify the issue.
Thanks!

**Comment by poojapatil36** [comment]
Verified the issue in below build.
1.49.0-insider(user setup)
Issue is fixed, hence closing the bug.
Please find the below attachment.
![20777_Issue is fixed](https://github.com/microsoft/azuredatastudio/assets/97668775/29083663-8e43-4741-be61-375a222596bb)
